apache/fineract-cn-mobile
# Feat: Add UI/Integration tests for creating group, group list and group details

Fixes #Issue_number

Please Add Screenshots If any UI changes.

**1. Test creating a group item successfully**
- Fill group details
- Add a member
- Add a leader
- Add address details
- Open review screen
- Click on the complete button
- Assert that snack bar message has been displayed

![createGroupSuccessfully](https://user-images.githubusercontent.com/49963168/92126731-29722f00-ee1e-11ea-97c7-0cba3b8a91b2.gif)

**2. Test creating a group item with invalid office name**
- Fill group details
- Enter office name with less than 5 character
- Click on NEXT button
- Assert that it doesn't allow navigating to add member fragment

![invalidOfficeName](https://user-images.githubusercontent.com/49963168/92126781-3858e180-ee1e-11ea-9d93-699ffb6bf61c.gif)

**3. Test group list item**
- Open navigation drawer
- Navigate to Group fragment
- Click on Group item with "group001" identifier
- Assert that group item details have been displayed correctly on GroupDetails screen

![groupListItem](https://user-images.githubusercontent.com/49963168/92126819-44dd3a00-ee1e-11ea-9ab1-e7df6a575ee5.gif)


Please make sure these boxes are checked before submitting your pull request - thanks!


- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything.

- [x] If you have multiple commits please combine them into one commit by squashing them.
